docker-compose.yml

```diff
@@ -6,20 +6,25 @@ networks:
   app:
     driver: bridge
     internal: false
-    
+
 services:
   mariadb:
     image: itkdev/mariadb:latest
     networks:
       - app
     ports:
       - '3306'
+    volumes:
+      - .docker/drupal/dumps:/docker-entrypoint-initdb.d
```

Comment:
You link here to a database dump, but the readme says that you have to do a `drush site-install`? It can't be both!

Reply:
Fixed